docs/getting_started/tutorial.rst

```diff
@@ -4,64 +4,121 @@ Your first Wagtail site
 .. note::
    This tutorial covers setting up a brand new Wagtail project. If you'd like to add Wagtail to an existing Django project instead, see :doc:`integrating_into_django`.
 
-1. Install Wagtail and its dependencies:
+Install and run Wagtail
+-----------------------
 
-   .. code-block:: console
+Install dependencies
+~~~~~~~~~~~~~~~~~~~~
+
+Wagtail supports Python 3.4, 3.5, 3.6, and 3.7.
+
+To check whether you have an appropriate version of Python 3:
+
+.. code-block:: console
+
+   $ python3 --version
+
+If this does not return a version number or returns a version before 3.4, you will need to `install Python 3 <https://www.python.org/downloads/>`_.
+
+.. important::
+   Before installing Wagtail, it is necessary to install the **libjpeg** and **zlib** libraries, which provide support for working with JPEG, PNG and GIF images (via the Python **Pillow** library).
+   The way to do this varies by platform—see Pillow's `platform-specific installation instructions <http://pillow.readthedocs.org/en/latest/installation.html#external-libraries>`_.
 
-       $ pip install wagtail
 
-2. Start your site:
+Create and activate a virtual environment
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-   .. code-block:: console
+We recommend using a virtual environment, which provides an isolated Python environment. This tutorial uses `venv <https://docs.python.org/3/tutorial/venv.html>`_, which is packaged with Python 3.
 
-       $ wagtail start mysite
-       $ cd mysite
+Create a virtual environment:
 
-   Wagtail provides a ``start`` command similar to
-   ``django-admin.py startproject``. Running ``wagtail start mysite`` in
-   your project will generate a new ``mysite`` folder with a few
-   Wagtail-specific extras, including the required project settings, a
-   "home" app with a blank ``HomePage`` model and basic templates and a sample
-   "search" app.
+.. code-block:: console
 
-3. Install project dependencies:
+   $ python3 -m venv mysite-env
```

Comment:
I directed venv to a different folder on the theory that it could be confusing to have the env things in the same folder as the project folder. Also because you don't really want your venv things in your wagtail project folder if you go to make your project a git repo.

Reply:
I do get your intention but I don't think the reasons are clear in the instruction. Your motivations can be reversed which makes an equally strong argument to keep the env in the project dir. Keeping it together is at least clear which env belongs to which project. I've seen more people mistake their env than mistakenly push their env to a repo.

One big advantage is that the env folder can be named `env`.

Git or CSV might be out of scope. But if you are concerned about it. You could add a line, `If you use a version control system, now is a good time to add `env` to ignore.`

Reply:
Changed to mysite/env and updated the instructions accordingly
https://github.com/wagtail/wagtail/pull/5210/commits/d79dddb666f8fe5216439088fa05c06eb52d179f